sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -1310,6 +1355,14 @@ struct _pi_ze_event_list_t {
     this->Length = other.Length;
     return *this;
   }
+
+  _pi_ze_event_list_t(const _pi_ze_event_list_t &other) {
+    this->ZeEventList = other.ZeEventList;
+    this->PiEventList = other.PiEventList;
+    this->Length = other.Length;
+  }
```

Comment:
this is physically duplicating the lists of events, right? why is that needed? and don't we end up with the same event being then referenced from multiple lists meaning we need to retain/release them in addition?

Reply:
These changes are redundant after patch update, reverted.